src/m365/spo/commands/sitescript/sitescript-get.ts

```diff
@@ -65,8 +69,13 @@ class SpoSiteScriptGetCommand extends SpoCommand {
         responseType: 'json'
       };
 
-      const res: any = await request.post(requestOptions);
-      await logger.log(res);
+      const response: any = await request.post(requestOptions);
+
+      if (args.options.content === true) {
+        return await logger.log(JSON.parse(response.Content));
+      }
+
+      await logger.log(response);
```

Comment:
it works but it's kinda strange (or uncommon is a better word) to `return` something (in this case `logger.log` which does not return anything) in a `void` method. Just seems confusing if someone does not know what `logger.log` is. 
either we should `return;` (so empty) or go with `if... else`

Reply:
Ok, good point, I'll fix that.